snippets/models.py

```diff
@@ -6,6 +6,15 @@
 from wagtail.snippets.models import register_snippet
 from openstax.functions import build_image_url
 
+
+BOOK_STATES = [
```

Comment:
Instead of defining this again...

`from books.models import BOOK_STATES`

Reply:
I tried that at first, but there is a circular reference that throws an error. Books imports Snippets, so Snippets cannot import Books.

Reply:
Ah, in that case, you might be able to change [this line](https://github.com/openstax/openstax-cms/pull/1186/files#diff-c1a435f4bd3b7dcd593e45dab6525331e8562ffb9f097a55bb99299b2fb2570aR168) to:

```
book_state = models.CharField(max_length=255, choices='books.BOOK_STATE', default='live', help_text='The state of the book.')
```

That's how you are able to do other references in Django when you run into circular import issues.

Reply:
I've not tested this on a choices field though

Reply:
What should really happen is all the foreign keys to snippets on books should change to `snippets.ModelName`. I'd like to avoid two definitions of book states - using either of those two options.

Reply:
I tried a multitude of solutions which did not work. I moved all the books model constants to a separate constants file and that works.